src/Jackett.Common/Indexers/GazelleGames.cs

```diff
@@ -268,69 +268,48 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                         var qNeutralLeech = Row.QuerySelector("strong.neutralleech_label");
-                        var RowTitle = Row.GetAttribute("title");
-
                         var Time = qTime.GetAttribute("title");
-                        release.PublishDate = DateTime.SpecifyKind(DateTime.ParseExact(Time, "MMM dd yyyy, HH:mm", CultureInfo.InvariantCulture), DateTimeKind.Unspecified).ToLocalTime();
-                        release.Category = GroupCategory;
-
-                        if (qDescription != null)
-                            release.Description = qDescription.TextContent;
+                        var guid = new Uri(SiteLink + qDLLink.GetAttribute("href"));
+                        var seeders = ParseUtil.CoerceInt(qSeeders.TextContent);
 
-                        release.Title = qDetailsLink.TextContent;
-                        release.Title = release.Title.Replace(", Freeleech!", "");
-                        release.Title = release.Title.Replace(", Neutral Leech!", "");
-
-                        if (stickyGroup) // AND match for sticky releases
-                            if ((query.ImdbID == null || !TorznabCaps.SupportsImdbMovieSearch) && !query.MatchQueryStringAND(release.Title))
-                                continue;
-
-                        var Size = qSize.TextContent;
-                        if (string.IsNullOrEmpty(Size)) // external links, example BlazBlue: Calamity Trigger Manual - Guide [GameDOX - External Link]
-                            continue;
-                        release.Size = ReleaseInfo.GetBytes(Size);
-
-                        release.Link = new Uri(SiteLink + qDLLink.GetAttribute("href"));
-                        release.Comments = new Uri(SiteLink + qDetailsLink.GetAttribute("href"));
-                        release.Guid = release.Link;
-
-                        release.Grabs = ParseUtil.CoerceLong(qGrabs.TextContent);
-                        release.Seeders = ParseUtil.CoerceInt(qSeeders.TextContent);
-                        release.Peers = ParseUtil.CoerceInt(qLeechers.TextContent) + release.Seeders;
-
-                        if (qFreeLeech != null)
-                        {
-                            release.UploadVolumeFactor = 1;
-                            release.DownloadVolumeFactor = 0;
-                        }
-                        else if (qNeutralLeech != null)
+                        releases.Add(new ReleaseInfo
                         {
-                            release.UploadVolumeFactor = 0;
-                            release.DownloadVolumeFactor = 0;
-                        }
-                        else
-                        {
-                            release.UploadVolumeFactor = 1;
-                            release.DownloadVolumeFactor = 1;
-                        }
-
-                        releases.Add(release);
+                            MinimumRatio = 1,
+                            MinimumSeedTime = 80 * 3600,
```

Comment:
replace with the multiplication with the result and let a comment with the time in hours